main.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'app'
+
+def prompt(message)
+  puts message
+  gets.chomp
+end
+
+def prompt_int(message)
+  prompt(message).to_i
+end
+
+def prompt_bool(message)
+  prompt(message).downcase == 'y'
+end
+
+def list_books(app)
+  app.list_all_books
+end
+
+def list_people(app)
+  app.list_all_people
+end
+
```

Comment:
- [ ] Please make sure that your list of people is working fine :pray: As you can see from the video below after I created two people and want to list them nothing returns and I think we should expect the list of people created :relaxed:  

_From the project requirement you should make sure that the app behaves in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)_
  
  [Screencast from 03-10-2023 08:34:22 PM.webm](https://user-images.githubusercontent.com/92303891/224425022-4b7cc8ee-5af5-463c-9223-02544387b75c.webm)

### :no_entry:  Since the list of people is not working, I couldn't check the functionality of the `_create_rental_` so the next code review should check if create rental and list rental are working as expected.